python/src/equistore/operations/equal_metadata.py

```diff
@@ -129,6 +129,86 @@ def equal_metadata(
     return True
 
 
+def equal_metadata_block(
+    block_1: TensorBlock, block_2: TensorBlock, check: Optional[List] = None
+) -> bool:
+    """
+    Checks if two :py:class:`TensorBlock` objects have the same metadata,
+    returning a bool.
+
+    If `check` is none (the default), all metadata (i.e. the samples,
```

Comment:
```suggestion
    If ``check`` is :py:obj:`None` (the default), all metadata (i.e. the samples,
```

All the check should have two backticks